Python/specialize.c

```diff
@@ -886,41 +884,33 @@ _Py_Specialize_LoadAttr(PyObject *owner, _Py_CODEUNIT *instr, PyObject *name, Sp
             break;
     }
     int err = specialize_dict_access(
-        owner, instr, type, kind, name, cache0,
+        owner, instr, type, kind, name,
         LOAD_ATTR, LOAD_ATTR_INSTANCE_VALUE, LOAD_ATTR_WITH_HINT
     );
     if (err < 0) {
         return -1;
     }
     if (err) {
-        if (_Py_OPCODE(instr[0]) == LOAD_ATTR_INSTANCE_VALUE) {
```

Comment:
This is a common pair and it saves a POP/PUSH pair and an incref/decref pair, so I'd like to revisit this in the future.
Do we have an issue for that?

Reply:
Discussion: https://github.com/faster-cpython/ideas/discussions/291
Issue: https://bugs.python.org/issue46823